content/docs/localization/localizing-addons.md

```diff
@@ -50,7 +50,7 @@ hungry2 = msg("eat", {food: msg("salmon")}) // shows "I want to eat salmon!"
 ```
 
 ### Safety
-If you are writing raw HTML, `msg` should be replaced with safer version of `safeMsg`.
+If you are writing raw HTML, `msg` should be replaced with safer version of `safeMsg`. Please note that `safeMsg` is not available in persistent scripts.
 
 ## Enabling localization
-Edit your `addon.json` to add `"l10n": true`. That's all you have to do!
\ No newline at end of file
+Edit your `addon.json` to add `"l10n": true`. That's all you have to do!
```

Comment:
Now that we're on it, this line (and paragraph title) can be removed

Reply:
done